releases/release-4.0.9.md

```diff
@@ -0,0 +1,103 @@
+---
+title: TiDB 4.0.9 Release Notes
+---
+
+# TiDB 4.0.9 Release Notes
+
+发版日期：2020 年 12 月 3 日
+
+TiDB 版本：4.0.9
+
+## 兼容性更改
+
++ TiDB
+
+- 废弃配置文件中 `enable-streaming` 配置项 [#21055](https://github.com/pingcap/tidb/pull/21055)
+
+## 新功能
+
++ TiFlash
+
+    - 支持将存储引擎的新数据分布在多个硬盘上，分摊 I/O 压力
+
+## 优化提升
+
++ TiDB
+
+    - 在转换等值条件为其它条件时，通过使用启发式规则，避免生成 (index) merge join 以得到更好的执行计划 [#21146](https://github.com/pingcap/tidb/pull/21146)
+    - 区分用户变量的类型 [#21107](https://github.com/pingcap/tidb/pull/21107)
+    - 配置文件中添加了 performance.gogc 配置项，用于设置 GOGC [#20922](https://github.com/pingcap/tidb/pull/20922)
+    - 提升了 `Timestamp` 和 `Datetime` 类型的二进制输出结果的 MySQL 兼容性 [#21135](https://github.com/pingcap/tidb/pull/21135)
+    - 对于可剪切的表达式进行常量折叠时，避免输出不必要的警告或错误 [#21040](https://github.com/pingcap/tidb/pull/21040)
+    - 优化了 `load data` 语句执行 prepare 操作的报错信息 [#21199](https://github.com/pingcap/tidb/pull/21199)
+    - 修改列类型的时候，忽略掉整型字段的零值填充大小属性 [#20986](https://github.com/pingcap/tidb/pull/20986)
+    - `explain analyze` 对于 DML 语句正确的显示执行器相关的运行时信息 [#21066](https://github.com/pingcap/tidb/pull/21066)
+    - 禁止一条语句中，对主键做出多次不同的修改 [#21113](https://github.com/pingcap/tidb/pull/21113)
+    - 添加了对于连接的空闲时间的监控项 [#21301](https://github.com/pingcap/tidb/pull/21301)
+
++ TiKV
+
+    -
```

Comment:
```suggestion
    - 添加标记以跟踪 split 命令的来源 [#8936](https://github.com/tikv/tikv/pull/8936)
    - 支持动态更改配置 `pessimistic-txn.pipelined` [#9100](https://github.com/tikv/tikv/pull/9100)
    - 减少运行 BR 和 Lightning 时对系统的性能影响 [#9098](https://github.com/tikv/tikv/pull/9098)
    - 添加关于 ingesting sst 报错的 metrics [#9096](https://github.com/tikv/tikv/pull/9096)
    - 如果还有副本在追日志，则停止进入休眠状态 [#9093](https://github.com/tikv/tikv/pull/9093)
    - 提高悲观锁流水线的成功率 [#9086](https://github.com/tikv/tikv/pull/9086)
    - 调整配置 `apply-max-batch-size` 和 `store-max-batch-size` 的值为 1024 [#9020](https://github.com/tikv/tikv/pull/9020)
    - 添加 `max-background-flushes` 配置 [#8947](https://github.com/tikv/tikv/pull/8947)
    - 对 storage 模块默认开启统一线程池 [#8887](https://github.com/tikv/tikv/pull/8887)
    - 默认关闭 Rocksdb consistency check 以提高性能 [#9029](https://github.com/tikv/tikv/pull/9029)
```

Reply:
added in https://github.com/pingcap/docs-cn/pull/4976/commits/1cd776f3a44c8d2e87b838a13efdaea88586ff0b.